### DISCUSSION TITLE
Output to PDF with logo

### DISCUSSION THREAD

**Discussion by nicolomonili** [chillerlan/php-qrcode#245] [Q&A] [answered]
Hi, 
always relating to this type of qrcode https://github.com/chillerlan/php-qrcode/discussions/244 , 
is it possible to get the same result, but in PDF format?

I'm doing some tests (i included the FPDF library), the PDF is generated correctly, but without the logo inside.

**Comment by codemasher** [comment, marked as answer]
The built-in PDF output has its limitations due to the underlying library, so you'd probably need to stick with the SVG output and [convert it to a raster image](https://github.com/chillerlan/php-qrcode/blob/v5.0.x/examples/imagickConvertSVGtoPNG.php) which you can then embed in PDF.
